x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/apm_signal_inventory/service_list/apm_services_table.tsx

```diff
@@ -56,9 +54,8 @@ import {
   SortFunction,
   TableSearchBar,
 } from '../../../../shared/managed_table';
-import { TryItButton } from '../../../../shared/try_it_button';
```

Comment:
Could this component be removed now that's not being used anymore?

Reply:
I think we could leave it there... we might have more features that will need it.